src/lib/init.ts

```diff
@@ -1,61 +1,58 @@
 import * as webnative from "webnative";
-import { setup } from "webnative";
-import type FileSystem from "webnative/fs/index";
 import { getRecoil, setRecoil } from "recoil-nexus";
 
 import { sessionStore, filesystemStore } from "../stores";
 import { SESSION_ERROR } from "../lib/session";
 import { getBackupStatus, type BackupStatus } from "../lib/auth/backup";
 
-// TODO: Add a flag or script to turn debugging on/off
-setup.debug({ enabled: false });
 
 const initialize = async (): Promise<void> => {
   try {
     let backupStatus: BackupStatus = null;
 
-    const state: webnative.AppState = await webnative.app({ useWnfs: true });
+    const program: webnative.Program = await webnative.program({
+      tag: { creator: "Fission", name: "WAT" },
+      debug: process.env.NODE_ENV === 'development',
```

Comment:
now tying `debug` directly to development mode. CRA ensures this is always exposed